lib/view_model.rb

```diff
@@ -276,6 +337,14 @@ def ==(other_view)
   def hash
     self.class._attributes.map { |attr| self.send(attr) }.hash
   end
+
+  private
+
+  def raise_access_check_error
```

Comment:
Sometimes I wish block parameters could be nicely named. This is one of those times. But yay overall at this function. How would you feel about moving the `raise` into the callers?